src/pages/projects/nowcasting.jsx

```diff
@@ -14,6 +14,17 @@ const NowcastingPage = ({ data }) => {
       <SEO title="Project: Nowcasting" />
       <h1>Nowcasting</h1>
       <main className="markdown">
+        <div className="videoWrapper">
+          <iframe
+            title="nowcasting"
+            width="560"
```

Comment:
I removed this locally and nothing changed - is this not needed anymore now?

Reply:
It was probably being cached by your probably I'd guess. But elements are needed.